src/ducks/Watchlists/Actions/Edit/EditForm/hooks.js

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useMemo } from 'react'
+import { useProjects } from '../../../../../stores/projects'
+export const useAllProjects = filter => {
```

Comment:
For function better to use the `function` declaration. We are using `const` for components, one line functions and variables